src/Umbraco.Web/Models/ContentEditing/ContentTypeBasic.cs

```diff
@@ -48,7 +48,7 @@ public string IconFilePath
             {
                 return IconIsClass
                            ? string.Empty
-                           : IOHelper.ResolveUrl("~/umbraco/images/umbraco/" + Icon);
+                           : IOHelper.ResolveUrl("images/umbraco/" + Icon);

```

Comment:
This change is incorrect. Any paths prefixed with ~/ are virtual paths relative to the virtual root... this is the purpose of the ~/ syntax and was created specifically for dealing with virtual directories. I will revert.